RNA-seq.wdl

```diff
@@ -93,6 +94,25 @@ workflow RNAseq {
 
     SampleConfig sampleConfig = read_json(convertSampleConfig.json)
 
+    # Create design matrix template
+    call shiny.CreateDesignMatrix as shinyDesign {
+        input:
+            countTable = expression.fragmentsPerGeneTable,
+            dockerImage = dockerImages["predex"],
+            shinyDir = outputDir + "/dgeAnalysis/"
+    }
+
+    # Create annotation file
+    if (defined(referenceGtfFile)) {
+        call shiny.CreateAnnotation as shinyAnnotation {
+            input:
+                referenceFasta = referenceFasta,
+                referenceGtfFile = select_first([referenceGtfFile]),
+                dockerImage = dockerImages["predex"],
+                shinyDir = outputDir + "/dgeAnalysis/"
+        }
+    }
+
```

Comment:
Maybe its a good idea to make these optional?
No doubt we will always be using them within SASC, since almost all RNA projects now also include the DGE analysis. But potential user outside of the LUMC might not need these files as standard output.